src/spikeinterface/sorters/external/herdingspikes.py

```diff
@@ -19,90 +16,72 @@ class HerdingspikesSorter(BaseSorter):
-        # extra probe params
-        "probe_masked_channels": "Masked channels",
-        "probe_inner_radius": "Radius of area around probe channel for localization",
-        "probe_neighbor_radius": "Radius of area around probe channel for neighbor classification.",
-        "probe_event_length": "Duration of a spike event (ms)",
-        "probe_peak_jitter": "Maximum peak misalignment for synchronous spike (ms)",
-        # extra detection params
-        "t_inc": "Number of samples per chunk during detection.",
-        "num_com_centers": "Number of centroids to average when localizing.",
-        "maa": "Minimum summed spike amplitude for spike acceptance.",
-        "ahpthr": "Requires magnitude of spike rebound for acceptance",
-        "out_file_name": "File name for storage of unclustered detected spikes",
-        "decay_filtering": "Experimental: Set to True at your risk",
-        "save_all": "Save all working files after sorting (slow)",
-        "amp_evaluation_time": "Amplitude evaluation time (ms)",
-        "spk_evaluation_time": "Spike evaluation time (ms)",
-        # extra pca params
-        "pca_ncomponents": "Number of principal components to use when clustering",
-        "pca_whiten": "If true, whiten data for pca",
-        # bandpass filter
-        "freq_min": "High-pass filter cutoff frequency",
-        "freq_max": "Low-pass filter cutoff frequency",
-        "filter": "Enable or disable filter",
-        # rescale traces
-        "pre_scale": "Scales recording traces to optimize HerdingSpikes performance",
-        "pre_scale_value": "Scale to apply in case of pre-scaling of traces",
-        # remove duplicates (based on spk_evaluation_time)
-        "filter_duplicates": "Remove spike duplicates (based on spk_evaluation_time)",
+        "localize": "Perform spike localization. (`bool`, `True`)",
+        "save_shape": "Save spike shape. (`bool`, `True`)",
+        "out_file": "Path and filename to store detection and clustering results. (`str`, `HS2_detected`)",
+        "verbose": "Print progress information. (`bool`, `True`)",
+        "chunk_size": " Number of samples per chunk during detection. If `None`, a suitable value will be estimated. (`int`, `None`)",
+        "common_reference": "Method for common reference filtering, can be `average` or `median` (`str`, `median`)",
+        "rescale": "Automatically re-scale the data.  (`bool`, `True`)",
+        "rescale_value": "Factor by which data is re-scaled. (`float`, `-1280.0`)",
+        "threshold": "Spike detection threshold. (`float`, `8.0`)",
+        "spike_duration": "Maximum duration over which a spike is evaluated (ms). (`float`, `1.0`)",
+        "amp_avg_duration": "Maximum duration over which the spike amplitude  is evaluated (ms). (`float`, `0.4`)",
+        "min_avg_amp": "Minimum integrated spike amplitude for a true spike. (`float`, `1.0`)",
+        "AHP_thr": "Minimum value of the spike repolarisation for a true spike. (`float`, `1.0`)",
```

Comment:
```suggestion
        "AHP_thr": "Minimum value of the spike repolarisation for a true spike. (`float`, `0.0`)",
```

The default of `AHP_thr` is `0.0`,